dsperse/src/server/handlers.py

```
@@ -53,99 +55,100 @@ async def handle_start_incremental_run(ctx: ServerContext, req: dict) -> dict:
     run = ctx.state.create_run(circuit_id, inputs, run_source, max_tiles)
     run.status = RunStatus.RUNNING
 
-    with tempfile.TemporaryDirectory() as tmp_dir:
+    data_dir = RUN_DATA_DIR / run.run_uid
+    data_dir.mkdir(parents=True, exist_ok=True)
+
+    asyncio.ensure_future(_process_run(ctx, run, inputs, slices_dir, data_dir))
```

Ruff (0.15.1)

[warning] Store a reference to the return value of `asyncio.ensure_future`

(RUF006)